atom/browser/api/atom_api_browser_window.cc

```diff
@@ -4,6 +4,7 @@
 
 #include "atom/browser/api/atom_api_browser_window.h"
 
+#include "atom/browser/api/atom_api_web_contents_view.h"
```

Comment:
Is this header needed? I don't think WebContentsView is used in this file

Reply:
It was accidentally added, will remove it.